ci/docker_acceptance_tests.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+set -e
+set -x
+
+# Since we are using the system jruby, we need to make sure our jvm process
+# uses at least 1g of memory, If we don't do this we can get OOM issues when
+# installing gems. See https://github.com/elastic/logstash/issues/5179
+export JRUBY_OPTS="-J-Xmx1g"
+export GRADLE_OPTS="-Xmx4g -Dorg.gradle.daemon=false -Dorg.gradle.logging.level=info -Dfile.encoding=UTF-8"
+
+SELECTED_TEST_SUITE=$1
```

Comment:
Just to improve readability, could we add comments here defining the options this variable can take? e.g.:

# Variable options:
# oss - tests oss
# full - tests full
# * - tests all